Admin/PageAdmin.php

```diff
@@ -45,6 +51,7 @@ protected function configureListFields(ListMapper $listMapper)
     protected function configureFormFields(FormMapper $formMapper)
     {
         parent::configureFormFields($formMapper);
+        $ivoryCKeditorAvailable = class_exists('Ivory\CKEditorBundle\IvoryCKEditorBundle') && $this->ivoryCkeditor;
```

Comment:
but if you prefer silent degration .. you can still check in the DI extension and then simply choose to ignore the config if the Bundle isn't installed